docs/API/components/list-from-data-source.md

```diff
@@ -89,6 +89,22 @@ Fliplet.Hooks.on('flListDataBeforeGetData', function onBeforeGetData(data) {
 });
 ```
 
+## Run a hook before deleting an entry
+```js
+Fliplet.Hooks.on('flListDataBeforeDeleteEntry', function onBeforeDeleteEntry(data) {
+  // data - (Object) Contains "entryId", "config", "container"
+  // data.entryId - (Number) ID of the entry that is going to be deleted
+  // data.config - (Object) Entire configuration object of the component
+  // data.container - (Object) DOM element of the container of the component
+
+  data.config.deleteData = function(id) {
+    // Resolve Promise
+    // Always resolve entry ID
```

Comment:
always resolve entry id? 🤔 let's make this more readable

Reply:
Vodka